app/components/posting-list.js

```diff
@@ -21,7 +21,7 @@ export default class PostingList extends React.PureComponent {
             <FlatList ListHeaderComponent={errorHeader}
                       ListFooterComponent={errorFooter}
                       data={props.postings}
-                      keyExtractor={item => item.id}
+                      keyExtractor={item => item.id.toString()}
```

Comment:
Can't the key extractor handle number values? Or are we handling something like `null` or `undefined`

Reply:
I'm unsure about null or undefined. React changed and now requires this to be string. Having this as number did lead to error messages in the log.